src/Wallet.purs

```diff
@@ -192,6 +200,6 @@ foreign import _submitTxNami
   -> Effect (Promise String) -- Submitted tx hash
 
 foreign import _attachSignature
-  :: ByteArray -- CBOR bytes of tx
-  -> ByteArray -- CBOR bytes of witness set
-  -> Effect (ByteArray)
+  :: CborBytes -- CBOR bytes of tx
+  -> CborBytes -- CBOR bytes of witness set
+  -> Effect (CborBytes)
```

Comment:
Just a stylistic nitpick but the parenthesis are redundant, I appreciate this wasn't your code.